Source/WebCore/PAL/pal/PALSwift/UnsafeOverlays.swift

```diff
@@ -37,5 +37,67 @@ extension HashFunction {
                 start: data.__dataUnsafe(), count: data.size()))
     }
 }
+extension CryptoKit.Digest {
+    public func toVectorUInt8<T: HashFunction>(_: T.Type) -> VectorUInt8 {
+        var rv = VectorUInt8(T.Digest.byteCount)
+        var index = 0
+        self.forEach { val in
+            rv[index] = val
+            index += 1
+        }
+        return rv
+    }
+
+}
+
+extension Data {
+    public static func fromSpan(spanNoCopy: SpanConstUInt8) -> Data {
+        return Data(
+            bytesNoCopy: UnsafeMutablePointer(mutating: spanNoCopy.__dataUnsafe()),
+            count: spanNoCopy.size(), deallocator: .none)
+    }
+}
+
+extension SpanConstUInt8: ContiguousBytes {
+    public func withUnsafeBytes<R>(_ body: (UnsafeRawBufferPointer) throws -> R) rethrows
+        -> R
+    {
+        try body(UnsafeRawBufferPointer(start: self.__dataUnsafe(), count: self.size()))
+    }
+}
+
+private class _CompileBugDontUse {
+    // rdar://116406681
+    private func forceLinkageForVectorDestructor() {
+        let _ = VectorUInt8()
+    }
+}
+
+extension SymmetricKey {
+    public func toVectorUInt8() -> VectorUInt8 {
+        return self.withUnsafeBytes { buf in
+            var rv = VectorUInt8(buf.count)
+            for index in 0..<buf.count {
+                rv[index] = buf.load(fromByteOffset: index, as: UInt8.self)
+            }
+            return rv
+        }
+
+    }
+}
```

Comment:
Same comment here about toVectorUInt8. 

Also, if this is going to happen more than once, you can probably make a generic function that accepts any ContiguousBytes thingy, and copies to vector.

Side point: We prefer to use "to" to indicate conversion, and "copyTo" to indicate a copy. Let's call this "copyTo".